test/archethic_test.exs

```diff
@@ -412,27 +412,24 @@ defmodule ArchEthicTest do
       assert :sys.get_state(dummy_poller_pid) == ArchEthic.Metrics.Helpers.poller_default_state()
 
       # create a dummy client
-      Process.flag(:trap_exit, true)
-      dummy_client_pid  =  spawn(fn ->
-            ArchEthic.Metrics.Poller.monitor()
-                loop()
-            end)
+      dummy_client_pid  =  Task.start(fn ->  ArchEthic.Metrics.Poller_dummy.monitor
+       loop()    end)
```

Comment:
I  did add here. , but I am confused , that it requires to start Poller with different name , but how to contact that same dummy poller to invoke its method monitor